### PR TITLE
fix/001 - add user last name and first name field

### PR DESCRIPTION
**Pull request by JerryZhao1025**
add user last name and first name field, as well as rename "user" table into "users"

### CODE REVIEW

File: src/main/java/com/laioffer/trademarket/entity/User.java [hunk 1]
```diff
@@ -5,15 +5,16 @@
...
 public class User implements Serializable {
 
     private static final long serialVersionUID = 2652327633296064143L;
 
     @Id
-    private String username;
-    private String password;
     private String email;
+    private String lastname;
+    private String firstname;
```
Comment (xincheng121) [inline src/main/java/com/laioffer/trademarket/entity/User.java:18]: 我觉得full name 或者 nickname 也可以

Comment (xincheng121) [reply src/main/java/com/laioffer/trademarket/entity/User.java:18]: 就是这么一说。没问题

**Review by xincheng121 — DISMISSED** [review]

**Comment by JerryZhao1025** [comment]
我上两个commits合起来干的事情i加了username和historyOrderList，然后他们的getter setter

**Review by xincheng121 — APPROVED** [review]

**Comment by xincheng121** [comment]
looks good!

**Pull request merged**